.travis.yml

```diff
@@ -12,6 +12,7 @@ cache:
   directories:
     - vendor
     - $HOME/.composer/cache
+    - $HOME/.phpbrew
```

Comment:
Actually, I'm simplifying this so it works as before, with a phpunit-bin directory and a single `phpunit` executable file in there. Makes things much easier to follow, and we don't need our shim.